EIPS/eip-223.md

```diff
@@ -0,0 +1,389 @@
+---
+eip: 223
+title: 223 Token with communication model
+description: Token with event handling and communication model
+author: Dexaran (@Dexaran)
+discussions-to: https://ethereum-magicians.org/t/erc-223-token-standard/12894
+status: Review
+type: Standards Track
+category: ERC
+created: 2017-05-03
+---
+
+## Abstract
+
+The following describes standard functions a token contract and contract which is intended to work with specified token can implement. This standard introduces a communication model which allows for the implementation of **event handling** on the receiver's side.
+
+## Motivation
+
+1. This token introduces a communication model for contracts that can be utilized to straighten the behavior of contracts that interact with tokens as opposed to [ERC-20](./eip-20.md) where a token transfer could be ignored by the receiving contract.
+2. This token is more gas-efficient when depositing tokens to contracts.
+3. This token allows for `_data` recording for financial transfers.
+
+
+## Specification
+
```

Comment:
Generally we recommend using the [blurb from the EIP template](https://github.com/ethereum/EIPs/blob/454b40fd2d9b788f0c9d21b58146b0a80b098ab2/eip-template.md?plain=1#L55) referring to RFC 2119 to define "MUST", "SHOULD", etc.